src/main/java/ToDo.java

```diff
@@ -0,0 +1,18 @@
+public class ToDo extends Task {
+
+    public ToDo(String description) {
+        super(description);
+    }
+
+    @Override
+    public String getType() {
+        return ("T");
```

Comment:
perhaps use a private variable to store the type information and return it instead? This way, it's easier to change the type's letter in one place rather than ctrl-F and searching throughout the code